packages/http-client-python/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Other Changes
 
+- Rename `apiview_mapping_python.json` cross-language id file to `apiview-properties.json` for cross-language compatibility
```

Comment:
@iscai-msft The log is added in wrong position since we have released 0.6.7 before this PR. So it shall be added for new version `0.6.8`